### ISSUE TITLE
[CONTENT] - Change header text on Newsletter component on `/library` page

### ISSUE THREAD

**Issue by bskinn** [Quansight/Quansight-website#237]
### What site is this for?

Quansight LLC

### Actual behavior

Current header text on the newsletter subscribe button on `/library` page is "Stay informed with our newsletter".

![image](https://user-images.githubusercontent.com/11325439/174675521-d2d9133d-8ec1-4085-99ef-8204faf45b39.png)

**Change to (including capitalization):** "Subscribe for Updates &amp; News"

(This is a stopgap for the more general fix in #238.)

**Comment by kherma** [comment]
Hi just to confirm, as #237  is area: lunch, that one should be done first, and then #238 as it's low priority. Is that correct?

**Comment by bskinn** [comment]
@kherma Yes, that's right. For launch, all that is needed is a change to the hard-coded text (#237). That will leave more time to address other, higher-priority issues.

Implementing the ability for us to modify that text in Storyblok (#238) is low priority, and can come after launch.
